src/general/OptionsMenu.cs

```diff
@@ -732,6 +745,21 @@ private void UpdateResetSaveButtonState()
         saveButton.Disabled = result;
     }
 
+    private void UpdateDefaultOutputDeviceText(OptionButton optionButton)
+    {
+        optionButton.SetItemText(0, TranslationServer.Translate("DEFAULT_OUTPUT_DEVICE"));
```

Comment:
Does it actually work if you have the item in the dropdown with this text and then having this text in en.po manually?
I'd be interested to know as this is important to know regarding: https://github.com/Revolutionary-Games/Thrive/issues/1870

Reply:
the way this works is:
The C# list uses the untranslated names (therefore "Default"). When adding the items to the godot dropdown I add the translated value instead of the real list value.

The updating does not use the text but rather the index of the selected item, therefore I can lookup the index in my C# list and get the untranslated value. You could probably generalize this logic a bit to have translatable drop down menus.

Reply:
I suppose this is a special case, yes, as the content is dynamically generated...